ios/FluentUI.Demo/FluentUI.Demo/DemoListViewController.swift

```diff
@@ -21,6 +21,8 @@ class DemoListViewController: UITableViewController {
         let demoListViewController = DemoListViewController(nibName: nil, bundle: nil)
 
         let navigationController = UINavigationController(rootViewController: demoListViewController)
+        navigationController.navigationBar.isTranslucent = true
```

Comment:
any impact on ios 14 and 13?

Reply:
Nope, looks the same as always on iOS 13 and 14.
![Simulator Screen Shot - iPhone SE (2nd generation) - 2021-10-01 at 15 29 23](https://user-images.githubusercontent.com/4934719/135692641-e655f3a7-f884-4614-9623-295254510d50.png)